lib/src/messages/responses/responses_base.dart

```diff
@@ -10,3 +10,10 @@ class ErrorResponseBase extends AbstractExceptionResponseBase
     return "[${super.errorCode}] ${super.message}";
   }
 }
+
+class NonErrorResponseBase {
```

Comment:
Would you be opposed to renaming this `ResponseBase`? It's more consistent with the other SDKs and since the only other class in the file is `ErrorResponseBase` the "NonError" part is already apparent.